src/cloudant/design_document.py

```diff
@@ -56,6 +57,17 @@ def views(self):
         """
         return self.get('views')
 
+    @property
+    def search_indexes(self):
+        """
+        Provides an accessor property to the indexes dictionary in the
+        locally cached DesignDocument.
+
+        :returns: Dictionary containing search index names and search functions
+            as key/value
+        """
+        return self.get('indexes')
```

Comment:
@ricellis you said: "If this is retrieving the indexes dictionary entry there is no guarantee they will only be search indexes is there? So shouldn't it just be indexes?"

Query indexes are not handled in the `DesignDocument` class.  I think it may be misleading to change this to `indexes` since we are only supporting search indexes.